content/sensu-go/6.0/operations/control-access/auth.md

```diff
@@ -957,6 +1007,15 @@ example      | {{< code shell >}}
 }
 {{< /code >}}
 
+<a name="ad-memberof"></a>
+
+| memberOf |    |
```

Comment:
Sorry for the confusion, the `memberOf` attribute is not a configuration attribute, but really an attribute of the user object returned by Active Directory; there's no configuration required apart from removing the `group_search` attribute. Makes sense?

Reply:
Thank you! I took another crack at it. Wdyt?